mmv1/products/spanner/BackupSchedule.yaml

```diff
@@ -0,0 +1,147 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'BackupSchedule'
+base_url: projects/{{project}}/instances/{{instance}}/databases/{{database}}/backupSchedules
+self_link: 'projects/{{project}}/instances/{{instance}}/databases/{{database}}/backupSchedules/{{name}}'
+create_url: 'projects/{{project}}/instances/{{instance}}/databases/{{database}}/backupSchedules?parent=projects/{{project}}/instances/{{instance}}/databases/{{database}}&backup_schedule_id={{name}}'
```

Comment:
Interesting, do we need the parent here too? I'm surprised it needs to be added as a query param since it is already part of the path.

Reply:
Removed parent